tests/components/lamarzocco/conftest.py

```diff
@@ -128,14 +128,6 @@ def mock_lamarzocco(device_fixture: MachineModel) -> Generator[MagicMock]:
         lamarzocco.firmware[FirmwareType.GATEWAY].latest_version = "v3.5-rc3"
         lamarzocco.firmware[FirmwareType.MACHINE].latest_version = "1.55"
 
-        async def websocket_connect_mock(
-            notify_callback: Callable | None,
-        ) -> None:
-            """Mock the websocket connect method."""
-            return None
-
-        lamarzocco.websocket_connect = websocket_connect_mock
-
```

Comment:
Thanks to autospec, `lamarzocco.websocket_connect` is already automatically defined as an `AsyncMock`